DesktopUI2/DesktopUI2/ViewModels/SavedStreamViewModel.cs

```diff
@@ -172,36 +171,52 @@ public void ViewOnlineSavedStreamCommand()
       //to open urls in .net core must set UseShellExecute = true
       Process.Start(new ProcessStartInfo(Url) { UseShellExecute = true });
       Tracker.TrackPageview(Tracker.STREAM_VIEW);
-
     }
 
     public void CopyStreamURLCommand()
     {
       Avalonia.Application.Current.Clipboard.SetTextAsync(Url);
       Tracker.TrackPageview("stream", "copy-link");
-
     }
 
     public async void SendCommand()
     {
       Progress = new ProgressViewModel();
       Progress.IsProgressing = true;
+
+      var dialog = Dialogs.SendReceiveDialog("Sending...", this);
+
+      _ = dialog.ShowDialog(MainWindow.Instance).ContinueWith(x =>
+      {
+        if (x.Result.GetResult == "cancel")
+          Progress.CancellationTokenSource.Cancel();
+      });
+
```

Comment:
Hey @clairekuang I was thinking we could add a X button to cancel the operation inside the circular progress bar instead of showing a dialog. WDYT?

Reply:
Hmm I think that would be better, but then we should try to align the quick send/receive button behavior as well